packages/node_modules/@cerebral/router/README.md

```diff
@@ -370,10 +388,9 @@ router.addRoutes([
 
 When used together with code splitting and `controller.addModule()` you can dynamically add functionality to a running cerebral application.
 
-The only gotcha is that you might need to refresh the current route when reloading the app. Due to the order of evens, the router may fire before the routes have loaded, This can be handled with an action that should be called after the `addRoutes` method has been called:
+The only gotcha is that you might need to refresh the current route when reloading the app. Due to the order of evens, the router may fire before the routes have loaded, This can be handled with an action that should be called after the `addRoutes` method has been called (or just use the router's `reload` operator in your signal):
 ```js
```

Comment:
Maybe remove the action and just propose the operator. It's the same but simpler.